addons/cargo/stringtable.xml

```diff
@@ -284,7 +284,7 @@
         </Key>
         <Key ID="STR_ACE_Cargo_space_edenDesc">
             <English>The cargo space available in this vehicle/container</English>
-            <German>Verfügbarer Frachtraum für Fahrzeuge und Container</German>
+            <German>Verfügbarer Frachtraum in diesem Fahrzeuge/Container</German>
```

Comment:
Fahrzeug

Reply:
?

Reply:
`in diesem FahrzeugE`. Sollte Einzahl sein.

Reply:
facepalm ^^